extensions/applicationinsights-debugplugin-js/scripts/listCdnVersions.ps1

```diff
@@ -0,0 +1,577 @@
+param (
```

Comment:
Mostly a copy from the AISKU -- changed to support only the ext/ folder and ai.dbg. prefix